README.md

```diff
@@ -37,14 +38,19 @@ If your Linux bootloader is Grub, you can follow these steps:
 
   * Edit /etc/default/grub (e.g. `sudo gedit /etc/default/grub`)
   * Add `intel_pstate=disable` to `GRUB_CMDLINE_LINUX_DEFAULT`
+  * Add `isolcpus=x,y,z` to `GRUB_CMDLINE_LINUX_DEFAULT`, where 'x,y,z' is a
+    comma separated list of all logical CPUs apart from the boot processor
+    (i.e. CPU 0). This ensures that the adaptive tick cores are used soley for
+    benchmarks (see 'Tickless Mode Linux Kernel').
   * Run `sudo update-grub`
 
 ### Tickless Mode Linux Kernel
 
 On a Linux system, Krun will insist that the kernel is running in
-"full tickless" mode. Tickless mode is a compile time kernel parameter, so if it is
-not enabled, you will need to build a custom kernel.  You can verify the
-tickless mode of the current kernel with:
+"full tickless" mode with the NO_HZ_FULL_ALL compile time flag. This will place
```

Comment:
This is great, but can we add a simple sentence that explains what `tickless mode` means?